onnxruntime/test/perftest/command_args_parser.cc

```diff
@@ -127,6 +127,8 @@ namespace perftest {
         test_config.run_config.inter_op_num_threads = static_cast<int>(OrtStrtol<PATH_CHAR_TYPE>(optarg, nullptr));
         if (test_config.run_config.inter_op_num_threads < 0) {
           return false;
+        } else {
+          test_config.run_config.enable_sequential_execution = false;
```

Comment:
Should parallel executor only get enabled when inter_op_num_threads > 1?

Reply:
If inter_op_num_threads=0, it uses the default std::max<int>(1, std::thread::hardware_concurrency() / 2);

so you mean we need to exclude inter_op_num_threads=1?

Reply:
The default value of 0 is kinda ambiguous with this change then. I think we meant to default to sequential executor, so maybe we should change inter_op_num_threads default value to 1?

---
In reply to: [331709085](https://github.com/microsoft/onnxruntime/pull/2000#discussion_r331709085) [](ancestors = 331709085)